pages/blogs/launch.ftd

```diff
@@ -0,0 +1,211 @@
+-- import: ui.fifthtry.com/components/page as p
```

Comment:
@nandhinidevie we have to put the blog UI in this package. The blog content has to be in ft repos' frontend folder.